examples/commands.reference.md

```diff
@@ -22,3 +22,4 @@
 | vscode.didact.view.refresh 	| Refresh the Didact Tutorials view. Called automatically each time vscode.didact.register is called. 	|  	|  	|  	| No output 	| https://github.com/redhat-developer/vscode-didact/wiki/Available-Commands-in-Didact-and-Elsewhere#vscodedidactviewrefresh 	|
 | vscode.didact.view.tutorial.open 	| Menu command used when the user right-clicks on a tutorial in the Didact Tutorials view to Start Didact Tutorial. 	| node: TreeNode / required 	|  	|  	| Check Didact output channel 	| https://github.com/redhat-developer/vscode-didact/wiki/Available-Commands-in-Didact-and-Elsewhere#vscodedidactviewtutorialopen 	|
 | vscode.didact.workspaceFolderExistsCheck 	| Simple check to see if the workspace has at least one root folder. 	| requirement: string / required 	|  	|  	| true on success, false on failure; result posted as message to update requirement element 	| https://github.com/redhat-developer/vscode-didact/wiki/Available-Commands-in-Didact-and-Elsewhere#vscodedidactworkspacefolderexistscheck 	|
+| vscode.didact.openNamedOutputChannel | Opens an output channel with the given name and also prints the given text to it if any is provided. | name: string / required | text: string / optional | | Check the named output channel | https://github.com/redhat-developer/vscode-didact/wiki/Available-Commands-in-Didact-and-Elsewhere#vscodedidactworkspacefolderexistscheck 	|
```

Comment:
It pretty much opens an outputchannel with the name provided. We do not care if it is there or not because VSCode lacks that api. We always create that channel. I opened a feature request at https://github.com/microsoft/vscode/issues/106993